searx/static/themes/oscar/src/js/01_init.js

```diff
@@ -32,3 +23,5 @@ window.searx = (function(d) {
         translations: JSON.parse(script.getAttribute('data-translations'))
     };
 })(document);
+
+window.searx = {};
```

Comment:
Temporary fix for `searx/static/themes/__common__/js/image_layout.js` : 
https://github.com/searxng/searxng/blob/a582cf3d8231f5ed8a881aa87576dfc0600e1c07/searx/static/themes/__common__/js/image_layout.js#L161

Reply:
> Temporary fix for `searx/static/themes/__common__/js/image_layout.js` ..

Fixes the issue / thanks!  .. could you have an eye on _"temporary"_

Reply:
`git grep -i "searx[^n]"` ?